vunit/ui/packagefacade.py

```diff
@@ -9,21 +9,33 @@
 """
 
 from pathlib import Path
+from typing import List, Optional, Union, TYPE_CHECKING
+from vunit.design_unit import DesignUnit
+
```

Comment:
```suggestion

from vunit.design_unit import DesignUnit
```
Not sure if any linting rules are applied, but this seems to make more sense either way.

Reply:
I do not really catch what your saying. But I did not find clear style re: import statements.
Also after doing all the changes I realized that vunit uses relative imports for most/all internal imports.
I personally do not like relative imports in python, but for consistencies sake I would change. If somebody from the core team can pitch in...

Reply:
Sorry for being unclear, the point was that if there is an empty line, it should probably not happen between the vunit imports.

Now
```
from typing...
from vunit...

from vunit...
```

Better
```
from typing...

from vunit...
from vunit...
```

(Tools like `isort` would probably have done something like the latter anyway.)

Reply:
Ah yes, thank you. Totally agreed.

(normally I just let isort handle imports)